main/Checkbox/index.tsx

```diff
@@ -138,8 +138,7 @@ const Checkbox: FC<CheckboxProps> = ({
             />}
           </Marker>
         </MarkerContainer>
-        <Label labelSize={labelSize} labelColor={isDisabled ? COLOR.LIGHTGRAY : labelColor}>
```

Comment:
Is there a reason for this subspace? I think the existing is better for readability!